service/src/main/java/uk/gov/hmcts/reform/fpl/service/CaseManagementOrderProgressionService.java

```diff
@@ -48,6 +58,7 @@ private void progressDraftCaseManagementOrder(CaseDetails caseDetails, CaseManag
             case SEND_TO_JUDGE:
                 caseDetails.getData().put(CASE_MANAGEMENT_ORDER_JUDICIARY.getKey(), order);
                 caseDetails.getData().remove(CASE_MANAGEMENT_ORDER_LOCAL_AUTHORITY.getKey());
+                publishReadyForJudgeReviewEvent(caseDetails, requestData);
```

Comment:
just curious, why not use `submitted` in  `ActionCaseManagementOrderController`?

Reply:
One for @swalker125

Reply:
I believe the reason was that as this handles the final flow of the CMO object (where it goes i.e. final -> draft, draft -> final...) it was decided to do it here instead, not 100% sure on that though